redis/src/main/scala/zio/redis/api/SortedSets.scala

```diff
@@ -2,7 +2,7 @@ package zio.redis.api
 
 import zio.duration._
 import zio.redis.Input._
-import zio.redis.Output._
+import zio.redis.Output.{ Tuple2Output, _ }
```

Comment:
```suggestion
import zio.redis.Output._
```

same here I think `_` is enough